File: create_dirs.py
#!/usr/bin/env python3
import os
import sys

dirs = [
    r'D:\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\AUTO CUAN\app\src\main\java\com\example\autocuanumkm\data\model',
    r'D:\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\AUTO CUAN\app\src\main\java\com\example\autocuanumkm\data\network',
    r'D:\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\AUTO CUAN\app\src\main\java\com\example\autocuanumkm\data\repository',
    r'D:\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\AUTO CUAN\app\src\main\java\com\example\autocuanumkm\ui\home',
    r'D:\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\AUTO CUAN\app\src\main\java\com\example\autocuanumkm\ui\navigation'
]

for directory in dirs:
    try:
        os.makedirs(directory, exist_ok=True)
        print(f'✓ Created: {directory}')
    except Exception as e:
        print(f'✗ Failed: {directory} - {e}')
        sys.exit(1)

print('\nAll directories created successfully!')

File: create_dirs.js
const fs = require('fs');

const dirs = [
  'D:\\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\\AUTO CUAN\\app\\src\\main\\java\\com\\example\\autocuanumkm\\data\\model',
  'D:\\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\\AUTO CUAN\\app\\src\\main\\java\\com\\example\\autocuanumkm\\data\\network',
  'D:\\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\\AUTO CUAN\\app\\src\\main\\java\\com\\example\\autocuanumkm\\data\\repository',
  'D:\\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\\AUTO CUAN\\app\\src\\main\\java\\com\\example\\autocuanumkm\\ui\\home',
  'D:\\VIOLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\\AUTO CUAN\\app\\src\\main\\java\\com\\example\\autocuanumkm\\ui\\navigation'
];

dirs.forEach(dir => {
  try {
    fs.mkdirSync(dir, { recursive: true });
    console.log('✓ Created: ' + dir);
  } catch (err) {
    console.error('✗ Failed: ' + dir + ' - ' + err.message);
  }
});

console.log('\nAll directories created successfully!');
